Source/ToolCore/Build/ResourcePackager.cpp

```diff
@@ -201,11 +201,6 @@ void ResourcePackager::GeneratePackage(const String& destFilePath)
             return;
         }
 
-        if (!file.GetSize())
-        {
-            return;
-        }
-
```

Comment:
Good thoughts, I was thinking about the folder approach too, maybe something like: "Include in all Builds", "Exclude from all Builds", "Include in Build" (with a combo box of what builds to include it in)

I'd like to look into how Unreal handles asset build tracking, Unity's being scene reference based... with a special folder for stuff to copy into build.  We can do some asset dependency stuff based on scene/prefab references, though I want customizable builds and to be able to drive assets solely from code... (I also want a code entry point and not a scene entry point, er, wait we already have that :stuck_out_tongue_winking_eye:) 

There should be a way to configure builds, much like in a compiler IDE, possible with preprocessor style tags, like "DEMO BUILD" or "GDC Presentation Build", or "SHIPPING", these tags could be applied to assets/folders to include in a build.  If you delete the build, then the tags would all automatically be removed, etc.

I guess the include/exclude/include in these platform builds, at a folder level, would be most expedient